drivers/hmis/app/models/hmis/group_viewable_entity.rb

```diff
@@ -53,5 +53,14 @@ class GroupViewableEntity < GrdaWarehouseBase
     scope :includes_entities, ->(entities) do
       where(id: Array(entities).flat_map { |entity| includes_entity(entity).pluck(:id) })
     end
+
+    scope :includes_any_entity_in_data_source, ->(data_source) do
+      project_ids = data_source.projects.select(:id)
+      organization_ids = data_source.organizations.select(:id)
+
+      where(entity: data_source).
+        or(where(entity_type: Hmis::Hud::Project.sti_name, entity_id: project_ids)).
+        or(where(entity_type: Hmis::Hud::Organization.sti_name, entity_id: organization_ids))
```

Comment:
I _think_ `GrdaWarehouse::ProjectCollectionMember` might make this simpler.  @ttoomey any chance you can take a quick look?

Reply:
I don't think we have the equivalent for HMIS, though it looks helpful. `GrdaWarehouse::ProjectCollectionMember` looks like it only has relationships to `Collection` which is the warehouse table

Reply:
makes sense.

Reply:
Hmm, project collection member is collection_id, project_id so not directly applicable for this query. Similar shape though